<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SCDTR_Interface</class>
 <widget class="QMainWindow" name="SCDTR_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SCDTR_Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="console_tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QCheckBox" name="plotjuggler_check_box">
            <property name="text">
             <string>Connect to plotjuggler</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="messages_check_box">
            <property name="text">
             <string>Show messages received</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextEdit" name="console"/>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="serial_port_labe">
              <property name="maximumSize">
               <size>
                <width>175</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>UDP server details</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="0">
               <widget class="QComboBox" name="udp_server_ip_combo"/>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="udp_server_port_combo">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="udp_server_ip_label">
                <property name="text">
                 <string>IP</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="udp_server_port">
                <property name="text">
                 <string>PORT</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="udp_server_connect">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Request_tab">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="5" column="0">
           <widget class="QPlainTextEdit" name="command_out"/>
          </item>
          <item row="4" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="luminaire_label">
              <property name="text">
               <string>Desired luminaire</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="extra_arg_line_edit"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="extra_arg_labels">
              <property name="text">
               <string>Extra arg (when applicable)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="luminaire_line_edit"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="command_in"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Commands to send:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="comment_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="send_cmd">
            <property name="text">
             <string>send_cmd</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
